src/pages/HomePage/HomePage.jsx

```diff
@@ -0,0 +1,49 @@
+import { Navbar, Sidebar } from "../../component";
+import "./homePage.css";
+import heroImg4 from "../../Assets/heroImg4.jpg";
+import breakfast from "../../Assets/breakfast.jpeg";
+import lunch from "../../Assets/lunch.jpg";
+import snacks from "../../Assets/snacks.jpg";
+import dinner from "../../Assets/dinner.jpg";
```

Comment:
you can have a index.js file in your assets folder and import all your images there. So that you can directly import a single file from assets

Reply:
i have one images.js file in assests but i don't know why it was not working i would be impleting it again in the next PR.
Thanks for the feedback